### PR TITLE
MAAS-1561 | Alamo check results endpoint

### PR DESCRIPTION
**Pull request by pawellesniewski**

### CODE REVIEW

File: vagrant/Vagrantfile [hunk 1]
```diff
@@ -25,4 +25,8 @@ Vagrant.configure(2) do |config|
   # your network.
   # config.vm.network "public_network"
 
+  config.vm.provider "virtualbox" do |v|
+    v.memory = 1024
+    v.cpus = 2
```
Comment (h00dy) [inline vagrant/Vagrantfile]: Please add also v.name

File: requirements.txt [hunk 2]
```diff
@@ -1,6 +1,9 @@
-Django>=1.8.4,<1.9
+cassandra-driver==3.0.0
+django-cassandra-engine==0.6.2
+django-cors-headers==1.1.0
 django-extensions
 django-filter==0.11.0
+Django>=1.8.4,<1.9
 djangorestframework==3.2.2
```
Comment (deejay1) [inline requirements.txt]: maybe 3.3.2?

**Comment by h00dy** [comment]
:+1:

**Comment by ch40s3k** [comment]
:+1:

**Pull request merged**